run “npm run build” failid in superset-frontend (0.37.0)

run “npm run build” failid in superset-frontend

Actual results
```
ERROR in ./src/visualizations/TimeTable/TimeTable.jsx 101:10
Module parse failed: Unexpected token (101:10)
You may need an appropriate loader to handle this file type, currently no loader
s are configured to process this file. See https://webpack.js.org/concepts#loade
rs
|       if (fullUrl) {
|         return (
>           <a href={fullUrl} rel="noopener noreferrer" target="_blank">
|             {column.label}
|           </a>

ERROR in ./src/SqlLab/App.jsx 111:2
Module parse failed: Unexpected token (111:2)
You may need an appropriate loader to handle this file type, currently no loader
s are configured to process this file. See https://webpack.js.org/concepts#loade
rs
|
| const Application = () => (
>   <Provider store={store}>
|     <ThemeProvider theme={supersetTheme}>
|       <App />

ERROR in ./src/explore/controls.jsx 135:23
Module parse failed: Unexpected token (135:23)
You may need an appropriate loader to handle this file type, currently no loader
s are configured to process this file. See https://webpack.js.org/concepts#loade
rs
|   includeTime: false,
|   description: t('One or many controls to group by'),
>   optionRenderer: c => <ColumnOption column={c} showType />,
|   valueRenderer: c => <ColumnOption column={c} />,
|   valueKey: 'column_name',

ERROR in ./src/explore/components/controls/VizTypeControl.jsx 146:6
Module parse failed: Unexpected token (146:6)
You may need an appropriate loader to handle this file type, currently no loader
s are configured to process this file. See https://webpack.js.org/concepts#loade
rs
|
|     return (
>       <div
|         className={`viztype-selector-container ${isSelected ? 'selected' : ''}
`}
|         onClick={this.onChange.bind(this, key)}

ERROR in ./src/dashboard/components/PropertiesModal.jsx 188:6
Module parse failed: Unexpected token (188:6)
You may need an appropriate loader to handle this file type, currently no loader
s are configured to process this file. See https://webpack.js.org/concepts#loade
rs
|     const { values, isDashboardLoaded, isAdvancedOpen } = this.state;
|     return (
>       <Modal show={this.props.show} onHide={this.props.onHide} bsSize="lg">
|         <form onSubmit={this.save}>
|           <Modal.Header closeButton>

ERROR in ./src/visualizations/FilterBox/FilterBox.jsx 233:26
Module parse failed: Unexpected token (233:26)
You may need an appropriate loader to handle this file type, currently no loader
s are configured to process this file. See https://webpack.js.org/concepts#loade
rs
|       }),
|     });
>     const options = (json?.data?.[key] || []).filter(x => x.id);
|     if (!options || options.length === 0) {
|       return [];

ERROR in ./src/components/TableSelector.jsx 243:6
Module parse failed: Unexpected token (243:6)
You may need an appropriate loader to handle this file type, currently no loader
s are configured to process this file. See https://webpack.js.org/concepts#loade
rs
|   renderDatabaseOption(db) {
|     return (
>       <span>
|         <Label bsStyle="default" className="m-r-5">
|           {db.backend}

ERROR in ./src/showSavedQuery/index.jsx 39:4
Module parse failed: Unexpected token (39:4)
You may need an appropriate loader to handle this file type, currently no loader
s are configured to process this file. See https://webpack.js.org/concepts#loade
rs
|
|   ReactDom.render(
>     <div>
|       <Form
|         schema={config.JSONSCHEMA}

ERROR in ./src/explore/controlPanels/FilterBox.jsx 41:9
Module parse failed: Unexpected token (41:9)
You may need an appropriate loader to handle this file type, currently no loader
s are configured to process this file. See https://webpack.js.org/concepts#loade
rs
|           },
|         ],
>         [<hr />],
|         [
|           {

ERROR in ./src/dashboard/index.jsx 42:16
Module parse failed: Unexpected token (42:16)
You may need an appropriate loader to handle this file type, currently no loader
s are configured to process this file. See https://webpack.js.org/concepts#loade
rs
| );
|
> ReactDOM.render(<App store={store} />, document.getElementById('app'));
|

ERROR in ./src/explore/controlPanels/Shared_DeckGL.jsx 42:2
Module parse failed: Unexpected token (42:2)
You may need an appropriate loader to handle this file type, currently no loader
s are configured to process this file. See https://webpack.js.org/concepts#loade
rs
|   'blob/master/superset-frontend/src/modules/sandbox.js';
| const jsFunctionInfo = (
>   <div>
|     {t(
|       'For more information about objects are in context in the scope of this
function, refer to the',

ERROR in ./src/explore/index.jsx 44:16
Module parse failed: Unexpected token (44:16)
You may need an appropriate loader to handle this file type, currently no loader
s are configured to process this file. See https://webpack.js.org/concepts#loade
rs
| );
|
> ReactDOM.render(<App store={store} />, document.getElementById('app'));
|

ERROR in ./src/components/TooltipWrapper.jsx 47:4
Module parse failed: Unexpected token (47:4)
You may need an appropriate loader to handle this file type, currently no loader
s are configured to process this file. See https://webpack.js.org/concepts#loade
rs
| }) {
|   return (
>     <OverlayTrigger
|       placement={placement}
|       overlay={<Tooltip id={`${kebabCase(label)}-tooltip`}>{tooltip}</Tooltip>
}

ERROR in ./src/components/ErrorBoundary.jsx 50:8
Module parse failed: Unexpected token (50:8)
You may need an appropriate loader to handle this file type, currently no loader
s are configured to process this file. See https://webpack.js.org/concepts#loade
rs
|       const firstLine = error.toString();
|       const message = (
>         <span>
|           <strong>{t('Unexpected error')}</strong>
|           {firstLine ? `: ${firstLine}` : ''}

ERROR in ./src/components/Button.jsx 54:4
Module parse failed: Unexpected token (54:4)
You may need an appropriate loader to handle this file type, currently no loader
s are configured to process this file. See https://webpack.js.org/concepts#loade
rs
|
|   let button = (
>     <BootstrapButton {...buttonProps}>{props.children}</BootstrapButton>
|   );
|   if (tooltip) {

ERROR in ./src/components/TableLoader.jsx 64:13
Module parse failed: Unexpected token (64:13)
You may need an appropriate loader to handle this file type, currently no loader
s are configured to process this file. See https://webpack.js.org/concepts#loade
rs
|   render() {
|     if (this.state.isLoading) {
>       return <Loading />;
|     }
|

ERROR in ./src/components/Menu/Menu.jsx 67:4
Module parse failed: Unexpected token (67:4)
You may need an appropriate loader to handle this file type, currently no loader
s are configured to process this file. See https://webpack.js.org/concepts#loade
rs
| }) {
|   return (
>     <StyledHeader className="top" id="main-menu">
|       <Navbar inverse fluid staticTop role="navigation">
|         <Navbar.Header>

ERROR in ./src/components/Select/OnPasteSelect.jsx 79:11
Module parse failed: Unexpected token (79:11)
You may need an appropriate loader to handle this file type, currently no loader
s are configured to process this file. See https://webpack.js.org/concepts#loade
rs
|   render() {
|     const { selectWrap: SelectComponent, ...restProps } = this.props;
>     return <SelectComponent {...restProps} onPaste={this.onPaste} />;
|   }
| }
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! superset@0.37.0 build: `cross-env NODE_OPTIONS=--max_old_space_size=819
2 NODE_ENV=production webpack --mode=production --colors`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the superset@0.37.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional log
ging output above.


Environment
(please complete the following information):

superset version: 0.36.0
python version: 3.6.10
node.js version: v12.18.3
npm version: 6.14.6
```

--- a/superset-frontend/webpack.config.js
+++ b/superset-frontend/webpack.config.js
@@ -340,6 +340,9 @@ const config = {
         exclude: [/superset-ui.*\/node_modules\//, /\.test.jsx?$/],
         include: [
           new RegExp(`${APP_DIR}/(src|.storybook|plugins|packages)`),
+          ...['./src', './.storybook', './plugins', './packages'].map(p =>
+            path.resolve(__dirname, p),
+          ), // redundant but required for windows
           /@encodable/,
         ],
         use: [babelLoader],

